src/main/java/frc/robot/commands/MoveToPose.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Default drive command. Drives robot using driver controls.
+ */
+public class MoveToPose extends CommandBase {
+
+  private final Drivetrain m_drive;
+  private Pose2d m_pose;
+  
+  public MoveToPose(Drivetrain drive, Pose2d pose) {
```

Comment:
This is already a command. It also factors in a relative transformation, to move a distance from the robot, rather than a global pose. I'm also not sure it's necessary in the Vision PR.

Reply:
This is for moving to a global pose, in front of the grids for scoring (using vision). There was no other command on the branch that does the same thing. I also changed it since then to only move to the selected spot.